polyglot-chat-app/README.md

```diff
@@ -1,47 +1,68 @@
 # Polyglot Chat Application
 
-This example demonstrates how to integrate Python on GraalVM with a Java application.
+This example demonstrates how to integrate Python on GraalVM with a Micronaut application.
 
 ### Prerequisites
 - [Native Image](https://www.graalvm.org/latest/reference-manual/native-image/)
-- [R support](https://www.graalvm.org/latest/reference-manual/r/)
 - [Python support](https://www.graalvm.org/latest/reference-manual/python/)
 
 ## Preparation
 
-1. Download and install the latest GraalVM JDK with Native Image, Python and R support using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader):
+1. Download the latest GraalPy as described on [https://www.graalvm.org/python/](https://www.graalvm.org/python/). For example on Linux:
     ```bash
-    bash <(curl -sL https://get.graalvm.org/jdk) -c 'python,R'
+    wget https://github.com/oracle/graalpython/releases/download/graal-23.1.1/graalpy-23.1.1-linux-amd64.tar.gz
```

Comment:
Could we use `pyenv` here? (If 23.1.1 was available?)

Reply:
Sure, but then we have to explain how to install pyenv to (presumably) Java users

Reply:
Ah, yes, point taken. Best to leave it as it is.